omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,130 @@
+Bulk imports
+============
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be named
+whatever you would like. It does not need to be placed in the folder
+from which the OMERO commands are run.
+
+A minimal YAML file might look like: ::
+
+    ---
+    path: "my-files.txt"
```

Comment:
This could be a bit confusing we start with ``.txt`` then quickly indicate ``.tsv`` and ``.csv`` below

Reply:
What would you suggest?

Reply:
either csv or tsv

Reply:
so like that users won't have to change extension when they start including some options described below

Reply:
For me, a file with one column doesn't immediately map to csv/tsv. For example, if I do:

```
find /my/data -name "*.dv" | tee my-files...
```

I would never think to name that csv or tsv.

Reply:
I can see your point.
Let's keep it as it is
The page can be updated later on if required